assets/javascripts/modules/attorneyBanner.js

```diff
@@ -34,4 +30,8 @@ module.exports = function(el) {
       idOfBanner.css('position', 'relative');
     }
   }
+
+  //Run the function to reposition the banner
+  setInterval(stickIt, 100);
```

Comment:
this is going to lead to poor performance. is it to make banner sticky? if so, consider using `$(window).scroll` or similar instead.